src/lib.rs

```diff
@@ -1,6 +1,7 @@
+#![deny(clippy::all, clippy::unwrap_used)]
 mod configuration;
 mod data;
-#[allow(renamed_and_removed_lints)]
+#[allow(renamed_and_removed_lints, clippy::unwrap_used)]
```

Comment:
So, wdyt? Should we remove it? Or shall we keep it?
It's more a question for the team tbh @didierofrivia @eguzki 
If we leave it, we need to all now _always_ really consider the error handling, which is probably a good thing, but leads to more work. If we remove it, I think we should redo this exercise regularly... 
Now leaving it doesn't mean use `.expect("I need this!")` instead of `.unwrap()` always obviously! :P

Reply:
I think it makes sense to remove it since we want fast iteration... we could start talking about code quality and practices once the storm passes. "Make it ugly, make it work" first, but we need to compromise to make it pretty and robust afterwards.

Reply:
I've removed the lint for now, we can re-enable this whenever we like, but should revisit this in any case